lib/overcommit/utils.rb

```diff
@@ -270,24 +270,12 @@ def broken_symlink?(file)
         Overcommit::Utils::FileUtils.symlink?(file) && File.size?(file).nil?
       end
 
-      # Convert a glob pattern to an absolute path glob pattern rooted from the
-      # repository root directory.
+      # Expand a list of relative globs to an array of absolute paths.
       #
       # @param glob [String]
-      # @return [String]
-      def convert_glob_to_absolute(glob)
-        File.join(repo_root, glob)
-      end
-
-      # Return whether a pattern matches the given path.
-      #
-      # @param pattern [String]
-      # @param path [String]
-      def matches_path?(pattern, path)
-        File.fnmatch?(pattern, path,
-                      File::FNM_PATHNAME | # Wildcard doesn't match separator
-                      File::FNM_DOTMATCH   # Wildcards match dotfiles
-        )
+      # @return [Array]
+      def expand_relative_glob(globs)
+        Array(globs).map { |glob| Dir[File.join(repo_root, glob)] }.flatten
```

Comment:
I should also mention that the options passed to `File.fnmatch?` are important, as I don't believe `Dir.glob` uses those options by default.